### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by jburel**
Add Changelog
and bump version ahead of new release

cc @hflynn

### CODE REVIEW

**Comment by hflynn** [comment]
You don't need to indent the bullet points, they are rendering as shaded code boxes at the moment. Content looks okay though

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# 0.3.0 (April 2016)
```
Comment (hflynn) [inline CHANGELOG.md]: 2017

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+0.3.0 (April 2016)
```
Comment (hflynn) [inline CHANGELOG.md]: 2017

**Review by hflynn — APPROVED** [review]

**Comment by jburel** [comment]
@hflynn happy with changes?

**Comment by hflynn** [comment]
Yep, looks good 👍

**Pull request merged**